packages/manager/apps/telecom/src/app/telecom/pack/migration/meeting/pack-migration-meeting.html

```diff
@@ -0,0 +1,135 @@
+                />
+                <label
+                    for="radioNoFakeMeeting"
+                    class="pl-2"
+                    data-translate="telecom_pack_migration_no_fake_meeting"
+                >
+                </label>
+            </div>
+        </div>
+        <div
+            class="alert alert-info"
+            role="alert"
+            data-ng-if="$ctrl.meetingSlots.canBookFakeMeeting && !$ctrl.meetingSlots.meetingSlots.length"
+            data-translate="telecom_pack_migration_no_erdv_info"
+        ></div>
+
+        <div
+            class="mt-3"
+            data-translate="telecom_pack_migration_contact_name_info"
+        ></div>
+
+        <div class="row">
+            <div class="form-group col-md-6">
+                <label
+                    class="control-label required"
+                    for="contactNameId"
+                    data-translate="telecom_pack_migration_contact_name"
+                ></label>
+                <input
+                    id="contactNameId"
+                    class="form-control"
+                    required
+                    pattern=".+"
+                    data-ng-model="$ctrl.contactName"
+                    placeholder="{{'telecom_pack_migration_contact_name'| translate}}"
+                />
+            </div>
+        </div>
+        <div class="row">
+            <div
+                class="col-md-6 mt-2 font-italic font-weight-lighter"
+                data-ng-show="$ctrl.meetingSlots"
+                data-translate="telecom_pack_migration_mandatory_field"
+            ></div>
+        </div>
+    </div>
+
+    <div
+        class="mt-3"
+        data-ng-show="!$ctrl.selectedMeeting.meetingSlots.slot"
+        data-translate="telecom_pack_migration_programmed_meeting_select"
+    ></div>
+    <div
+        class="mt-3"
+        data-ng-show="$ctrl.selectedMeeting.meetingSlots.slot && !$ctrl.selectedMeeting.meetingSlots.fakeMeeting"
+        data-ng-bind-html="$ctrl.meetingSelectMessage"
+    ></div>
+
+    <div class="row">
+        <div
+            class="card col-md-2 shadow-sm bg-white rounded"
+            data-ng-repeat="meeting in $ctrl.meetings track by $index"
+        >
+            <div class="card-body meeting">
+                <h5 class="card-title" data-ng-bind="meeting.title"></h5>
+                <oui-radio
+                    data-ng-repeat="slot in meeting.slots track by $index"
+                    data-on-change="$ctrl.selectSlot(slot.id)"
+                    value="slot.start"
+                    data-model="$ctrl.model"
+                >
+                    {{slot.startTime}} - {{slot.endTime}}
+                </oui-radio>
+            </div>
+        </div>
+    </div>
+</div>
+<div class="mt-5" data-ng-if="!$ctrl.loading.init">
+    <a
+        class="btn btn-default"
+        data-translate="telecom_pack_migration_confirm_cancel_button"
+        data-ui-sref="telecom.packs.pack"
```

Comment:
Question: Is telecom.packs.pack is accessible here ?

Reply:
yes it is